scripts/auto-rebase/rebase.sh

```diff
@@ -1141,6 +1145,64 @@ rebase_to() {
     rm -rf "${STAGING_DIR}"
 }
 
+update_last_lvms_rebase() {
+    local lvms_operator_bundle_manifest="$1"
+
+    title "## Updating last_lvms_rebase.sh"
+
+    local last_rebase_script="${REPOROOT}/scripts/auto-rebase/last_lvms_rebase.sh"
+
+    rm -f "${last_rebase_script}"
+    cat - >"${last_rebase_script}" <<EOF
+#!/bin/bash -x
+./scripts/auto-rebase/rebase.sh lvms-to "${lvms_operator_bundle_manifest}"
+EOF
+    chmod +x "${last_rebase_script}"
+
+    (cd "${REPOROOT}" && \
+         test -n "$(git status -s scripts/auto-rebase/last_lvms_rebase.sh)" && \
+         title "## Committing changes to last_lvms_rebase.sh" && \
+         git add scripts/auto-rebase/last_lvms_rebase.sh && \
+         git commit -m "update last_lvms_rebase.sh" || true)
+}
+
+# Runs each LVMS rebase step in sequence, commiting the step's output to git
+rebase_lvms_to() {
+    local lvms_operator_bundle_manifest="$1"
+
+    title "# Rebasing LVMS to ${lvms_operator_bundle_manifest}"
+
+    download_lvms_operator_bundle_manifest "${lvms_operator_bundle_manifest}"
+
+    # LVMS image names may include `/` and `:`, which make messy branch names.
+    rebase_branch="rebase-lvms-${lvms_operator_bundle_manifest//[:\/]/-}"
+    git branch -D "${rebase_branch}" || true
+    git checkout -b "${rebase_branch}"
+
+    update_last_lvms_rebase "${lvms_operator_bundle_manifest}"
+
+    update_lvms_images
+    if [[ -n "$(git status -s pkg/release)" ]]; then
+        title "## Committing changes to pkg/release"
+        git add pkg/release
+        git add packaging/crio.conf.d
```

Comment:
is this necessary to be updated in lvms rebase (I thought it would only touch assets)? Surprised to see CRIO/pkg/release configuration here

Reply:
No, that was a copy-and-paste error from duplicating the commit step. I'll clean that up in the next revision.

Reply:
Done